test/helpers/assert-tasks.js

```diff
@@ -1,7 +1,14 @@
 const assert = require('assert');
+const { difference } = require('lodash');
+const seeds = require('../data/tasks');
 
 module.exports = (expected, actual) => {
-  assert.equal(actual.length, expected.length, `Expected ${expected.length} records. Received ${actual.length}`);
   const names = actual.map(task => task.data.data.name);
-  names.forEach(name => assert(expected.includes(name), `Expected [${expected.join(', ')}] to include "${name}"`));
+  const all = seeds.map(task => task.data.data.name);
+  const nopes = difference(all, names);
+
+  names.forEach(name => assert(expected.includes(name), `[${expected.join(', ')}] should include "${name}"`));
+  nopes.forEach(name => assert(!names.includes(name), `[${actual.join(', ')}] should not include "${name}"`));
```

Comment:
This now calculates a list of all the tasks it should _not_ expect and checks they are not present, which is a better output than simply comparing the length.